extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -58,7 +64,7 @@
  * @author Stefan Bußweiler - Added new thing status handling
  * @author Jochen Hiller - fixed status updates, use reachable=true/false for state compare
  */
-public class HueBridgeHandler extends BaseBridgeHandler {
+public class HueBridgeHandler extends BaseBridgeHandler implements ConfigStatusProvider {
```

Comment:
We could extend the ConfigStatusThingHandler... Then the operations ```supportsEntity``` and ```setConfigStatusCallback``` are already implemented